### ISSUE TITLE
Add Code or scripts to destroy all created resources

### ISSUE THREAD

**Issue by 211217613** [MicrosoftDocs/azure-dev-docs#281]
How would someone go back and destroy all the resources created?

Can you add a script to nuke all created resources in this tutorial.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 059a1972-09ee-02a7-8159-07ba04a2c763
* Version Independent ID: 4323aa16-062c-1b11-fb73-11777c34f88e
* Content: [Tutorial - Configure dynamic inventories of your Azure resources using Ansible](https://docs.microsoft.com/en-us/azure/developer/ansible/dynamic-inventory-configure)
* Content Source: [articles/ansible/dynamic-inventory-configure.md](https://github.com/MicrosoftDocs/azure-dev-docs/blob/master/articles/ansible/dynamic-inventory-configure.md)
* Service: **ansible**
* GitHub Login: @TomArcherMsft
* Microsoft Alias: **tarcher**

**Comment by TomArcherMsft** [comment]
@211217613 Thanks for the feedback. There is no script required. All you need to do is delete the resource group *ansible-inventory-test-rg*. However, it's probably a good idea to include that in the article. I'll update the article. Thanks again.

**Comment by 211217613** [comment]
does deleting the resource group, delete all resources in the resource group?

**Comment by TomArcherMsft** [comment]
Yes. [az group delete](https://docs.microsoft.com/cli/azure/group?view=azure-cli-latest#az-group-delete)

**Comment by TomArcherMsft** [comment]
You can also [use an Ansible playbook to delete the group](https://docs.microsoft.com/azure/developer/ansible/application-gateway-configure#clean-up-resources)

**Comment by 211217613** [comment]
This is kind of a meta question but what is the benefit of using a playbook then passing in the resource group like `--extra-vars` as opposed to just running a `az` command?

**Comment by TomArcherMsft** [comment]
@211217613 If you're working interactively, you could use many different technologies (Az CLI, PowerShell Az module, Ansible, etc.)  However, since this is an Ansible article and readers will ostensibly migrate this code to a CI server, we wanted to show how to perform the task using an Ansible playbook so that the reader can see how the task would be coded.

**Comment by TomArcherMsft** [comment]
@211217613 

I like your ideas of using `--exta-vars` and I also like the idea of showing multiple ways to accomplish the task (deleting the resource group, in this case). 

Therefore, I've updated the Dynamic Inventory article (and many other articles) to show how to delete the resource group using Ansible, Azure Cli, and Azure PowerShell. I've also updated the sample Ansible playbook to be variable driven for use with `--extra-vars`.

I've updated the articles that only delete a resource group as their cleanup step. I'm going through the other articles and figuring out how to incorporate the same style of cleanup into them as well.

Thanks for the great feedback.
